xmr-btc/src/config.rs

```diff
@@ -24,7 +25,8 @@ impl Config {
             bob_time_to_act: *regtest::BOB_TIME_TO_ACT,
             bitcoin_finality_confirmations: regtest::BITCOIN_FINALITY_CONFIRMATIONS,
             bitcoin_avg_block_time: *regtest::BITCOIN_AVG_BLOCK_TIME,
-            monero_max_finality_time: *regtest::MONERO_MAX_FINALITY_TIME,
+            monero_max_finality_time: *regtest::MONERO_AVG_BLOCK_TIME
```

Comment:
The issue with this calculation is that it is used as an upper bound: if the time is passed then a party will assume that the other one is not proceeding.

We should apply a scaling factor (1.5 or 2) so that the swap is not aborted when the blockchain is slow